.eslintrc

```diff
@@ -43,14 +43,14 @@ rules:
   no-shadow: 2
   no-multi-spaces: 2
 
-  # WARNINGS
-  # This is the only one that's hard to track since we don't lint just changes.
-  max-len: [1, 80]
-
   # WISHLIST. One day...
   # We'll need a custom version of this that does a subset of the whole rule.
   # Otherwise this is just too noisy.
   # valid-jsdoc: 1
+  # Ideally, we could just warn when *new* lines are added that exceed 80 chars
+  # while not warning about existing ones (often URLs, etc. which are
```

Comment:
There's an issue to allow longer lines with urls here https://github.com/eslint/eslint/issues/1661 but no implementation  yet (would be the same the jscs option `allowUrlComments` http://jscs.info/rule/maximumLineLength.html)